pom.xml

```diff
@@ -335,25 +333,25 @@
             <dependency>
                 <groupId>io.confluent</groupId>
                 <artifactId>kafka-json-schema-provider</artifactId>
-                <version>${io.confluent.schema-registry.version}</version>
+                <version>7.3.0-615</version>
             </dependency>
 
             <dependency>
                 <groupId>io.confluent</groupId>
                 <artifactId>kafka-protobuf-provider</artifactId>
-                <version>${io.confluent.schema-registry.version}</version>
+                <version>7.3.0-615</version>
             </dependency>
 
             <dependency>
                 <groupId>io.confluent</groupId>
                 <artifactId>kafka-connect-json-schema-converter</artifactId>
-                <version>${io.confluent.schema-registry.version}</version>
+                <version>7.3.0-615</version>
             </dependency>
 
             <dependency>
                 <groupId>io.confluent</groupId>
                 <artifactId>kafka-connect-protobuf-converter</artifactId>
-                <version>${io.confluent.schema-registry.version}</version>
+                <version>7.3.0-615</version>
```

Comment:
Just to confirm, the common pom version and the kafka.version are different, right?

Reply:
Yes. The `kafka.version` is actually set in the common pom. This line is the schema registry version, though, and it's also different than the common pom.